src/main/java/dk/kb/present/View.java

```diff
@@ -84,10 +85,11 @@ public MediaType getMime() {
     }
 
     @Override
-    public String apply(String recordID, String content) {
+    public String apply(String recordID, String content, String relation) {
```

Comment:
Unless you means for the `relation` to be a placeholder for either `child` or `parent`, please name the variable more precisely.